megabeast/tools/make_naive_maps.py

```diff
@@ -79,7 +90,8 @@ def create_naive_maps(stats_filename, pix_size=10.0, verbose=False):
     for i in range(n_x + 1):
         for j in range(n_y + 1):
             (tindxs,) = np.where((x == i) & (y == j))
-            # tindxs, = np.where((x == i) & (y == j) & (cat['chi2min'] < 10.))
+            if chi2mincut:
+                (tindxs,) = np.where((x == i) & (y == j) & (cat['chi2min'] < chi2mincut))
```

Comment:
Would be better if line 92 was in an else statement after line 94.  Then there wouldn't be a double execution.

Reply:
Addressed with commit from May 23.